swagger/swagger-generator/generator-core/src/main/java/io/servicecomb/swagger/generator/core/processor/response/DefaultResponseTypeProcessor.java

```diff
@@ -25,6 +25,11 @@
 import io.swagger.models.properties.Property;
 
 public class DefaultResponseTypeProcessor implements ResponseTypeProcessor {
```

Comment:
How about name it with NullResponseTypeProcessor

Reply:
all not special type will use this processor to generate contract
so default is better.